sound/sound-font-config.ini-configuration.md

```diff
@@ -25,111 +27,206 @@ volHum=15
+#
+# ProffieOS.SFX.EFFECTNAME.volume=100
+ProffieOS.SFX.boot.volume=75
+
+# Make smoothswings start in sync with hum.
+# Set to 1 to sync, or 0 to resume swings where last pair left off.
+ProffieOSSmoothSwingHumstart=1
+
+# Pair sequential effects such as preon->out.wav files.
+# EFFECTNAME can be “preon”, “out”, "lock", “in”, etc.
+# Set to 1 *AND* have the same number of files in the second effect
+# to activate this feature. Only the first in the chain need be paired here.
+# ProffieOS.SFX.EFFECTNAME.paired=1
+# For example:
+# pair preon -> out.wavs
+#ProffieOS.SFX.preon.paired=1
+# pair in -> pstoff.wavs
+#ProffieOS.SFX.in.paired=1
+
+# ---- As of ProffieOS 7 ---- #
+# bgnlock -> lock
+#ProffieOS.SFX.bgnlock.paired=1
+# lock -> endlock
+#ProffieOS.SFX.lock.paired=1
+#ProffieOS.SFX.bgndrag.paired=1
+#ProffieOS.SFX.drag.paired=1
+#ProffieOS.SFX.bgnlb.paired=1
+#ProffieOS.SFX.lb.paired=1
+#ProffieOS.SFX.bgnmelt.paired=1
+#ProffieOS.SFX.melt.paired=1
+#ProffieOS.SFX.bgnarm.paired=1
+# armhum -> endarm (if disarmed) -> boom
+#ProffieOS.SFX.armhum.paired=1
+
+# out -> hum
+#ProffieOS.SFX.out.paired=1
+
+# hum -> in
+# *Note* When multiple hum files exist,
+# even if not the same number of files as in.wavs,
+# (so it won't pair to them,) this also will loop the 
+# same hum sound until next ignition instead of randomly 
+# choosing a different file once the current hum ends,
+#ProffieOS.SFX.hum.paired=1
+
+# Set smoothswings to start in sync with hum start.
+# Set to 1 to sync, or 0 to resume swings where last pair left off.
+# Default 0
+ProffieOSSmoothSwingHumstart=1
+
+# ---- OLED displays ---- #
+
+# Pair OLED images to sounds with same numbers
+# Ex: blst02.bmp shows when blst02.wav plays
+# If quantity of images=wavs, all files are paired 1:1.
+# If quantity of images<wavs, unmatched higher numbered wavs all use highest numbered image.
+# If quantity of images>wavs, unmatched images get ignored.
+# If not set to pair, images are randomly chosen, regardless of quantities of sounds/images.
+# ProffieOS.IMG.EFFECTNAME.paired=1
+#ProffieOS.IMG.boot.paired=1
+#ProffieOS.IMG.blst.paired=1
+#ProffieOS.IMG.clsh.paired=1
+#ProffieOS.IMG.lock.paired=1
+
+# This specifies the frame rate of animtions.
 # If not specified (or set to zero) the frame rate will be 24 frames
 # per second for non-looped animations. For looped animations, the
 # frame rate will be set so that the loop takes one second.
 ProffieOSAnimationFrameRate=0.0
 
-# For OLED displays, the time a blade style's "message", 
-# static BMP, or loop animation will play when changing presets.
-ProffieOSFontImageDuration=5000.0
+#-----------------
+# Below are the display durations.
+# If commented out or not specified, the default is used.
+# As of ProffieOS 7, effect IMAGES can use 0.0 to automatically 
+# sync to the effect sound file duration.
 
-# For OLED displays, the time an on.bmp will play
-ProffieOSOnImageDuration=5000.0
+# Text Message duration
+# Default -1.0
```

Comment:
When the default is -1, we need to explain what that actually means.

Reply:
Which is.....allowing it to be set to zero?